### PR TITLE
Create rectangle_cut_vector.cpp

### PR DESCRIPTION
**Pull request by ashu-cybertron**
rectangle cut using vector

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)
yes
### Description

(Write your answer here.)
this is the rectangle cutting program in CPP using vector
### Checklist

- [*] I've read the contribution guidelines.
- [*] I've checked the issue list before deciding what to submit.
- [*] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

**Comment by ashu-cybertron** [comment]
i have tried in Cpp using vector plz consider it as another code rather than one submitted earlier

File: C-Plus-Plus/dp/rectangle_cut_vector.cpp [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+int main() {
+    int a, b; std::cin >> a >> b;
+    vector<vector<int> > mem(a+1, vector<int>(b+1, -1));
+    int res = minimum_cuts(a,b,mem);
+    std::cout << res << std::endl;
+    return 0;
+}
+//
+//sample input : 3 5 
+//output : 3
```
Comment (vatsalkesarwani12) [inline C-Plus-Plus/dp/rectangle_cut_vector.cpp:38]: provide time and space complexity of the code.

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/dp/rectangle_cut_vector.cpp (hunk 1 above)
Comment (ashu-cybertron) [reply C-Plus-Plus/dp/rectangle_cut_vector.cpp:38]: updated time and space needed @plazzy99

**Comment by ashu-cybertron** [comment]
done

**Comment by HarshCasper** [comment]
Add the corresponding README changes as well.

**Review by ashu-cybertron — COMMENTED** [review]
added time and space complexity

**Comment by HarshCasper** [comment]
Add README changes as well

**Comment by HarshCasper** [comment]
@ashu-cybertron Are you still working on it?

**Comment by HarshCasper** [comment]
I'm closing the PR due to no activity. If you would like to reopen this PR to be reviewed please join the [Slack](https://join.slack.com/t/tesseractcodi-bxq5968/shared_invite/zt-ju6c0nqq-KgkTvFyxjNVbDRBIjADanw) and drop me a message.

**Pull request closed without merging**